server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/delete/DeleteSnapshotRequest.java

```diff
@@ -27,9 +28,12 @@
  */
 public class DeleteSnapshotRequest extends MasterNodeRequest<DeleteSnapshotRequest> {
 
+    public static final boolean DEFAULT_WAIT_FOR_COMPLETION = true;
```

Comment:
Just because we assign the default in the `DeleteSnapshotRequest` class and also when processing the `RestRequest`. Could reduce this duplication in a number of ways but this seemed the simplest.